### PR TITLE
ks - adding swagger option

### LINKED ISSUE #8: FEATURE: Swagger option on navbar
# User Story

* As a developer
* I can easily turn on the swagger option on the app navbar for any deployment through defining an environment variable `SHOW_SWAGGER_UI_LINK`
* so that it is easy to access swagger when I need it, but I can easily turn it off when I don't.

### Discussion

There is already code in the AppNavbar.js file that checks the value returned by /api/systemInfo for turning the Swagger link on and off.  But, at the moment, the resource file isn't set up to pull that value from an environment variable.

This PR shows what changes are needed to make this work:

https://github.com/ucsb-cs156/proj-courses/pull/93

Your job is just to make similar changes in this repo.

You could also make a similar issue that allows users to turn on/off the H2 console via a variable called `H2_CONSOLE_ENABLED`.  But it would be smart to do that in a separate PR so you get more points!

### Acceptance Criteria

As a developer
- [ ] When the config variable on dokku SHOW_SWAGGER_UI_LINK is set to true, the swagger link shows up
- [ ] When the config variable on dokku SHOW_SWAGGER_UI_LINK is set to false, the swagger link does NOT  show up.

### PR DESCRIPTION
**Pull request by kriteenshrestha**
In this PR, I changed the resource file, as the sample PR showed, so that we read from the the environment variable that controls the swagger link option.
image

Closes #8

### CODE REVIEW

**Review by evanja57 — DISMISSED** [review]
LGTM

**Review by WhrainySu — DISMISSED** [review]
LGTM

File: src/main/resources/application-development.properties [hunk 1]
```diff
@@ -5,8 +5,10 @@ spring.datasource.username=sa
 spring.datasource.password=password
 spring.h2.console.settings.web-allow-others=true
 spring.h2.console.enabled=true
-app.showSwaggerUILink=true
+# app.showSwaggerUILink=true
```
Comment (pconrad) [inline src/main/resources/application-development.properties]: Please remove the commented out line of code.

And team members: please do not put LGTM on a PR that has commented out lines in it.

**Review by pconrad — CHANGES_REQUESTED** [review]
Please see: https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html

**Review by evanja57 — APPROVED** [review]
LGTM! For real this time, no commented out code!

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**